shared/tests/utility/math/docs/genValsInRange.m

```diff
@@ -0,0 +1,6 @@
+function randVals = genValsInRange(min, max, total, cols)
+%This function generates random values in the given range
+%params: min = lower val, max = upper val, total = total numbers generated
+% cols = number of columns (use this for vectors etc)
+randVals = (max - min).*rand(total,cols) + min;
```

Comment:
If we're not giving matlab a random seed, won't this produce different values each time? I think it would be good if this set the seed, or at least commented, noting the seed which produced the values used in the final tests.

Reply:
Hmmm, yes. I didn't think of using a seed. I was only accounting for if someone wanted to regen a new set. The extra copies of the test data were included in the docs folder in case of an accidental deletion or something. If we want it to make the same data each time i'm happy to work on it further though.

Reply:
Since you have it committed to git, there's no need to worry about accidental deletion. And, now that a commit with the data been pushed to the remote repo (and thus to the github servers), the information will live on as long as the repo does, accessible basically forever, through that commit.